kusto-language-server/package.json

```
@@ -2,8 +2,8 @@
     "name": "kuskus-kusto-language-server",
     "displayName": "[Kuskus] Kusto Language Server",
     "description": "Language server for the Kusto query language",
-    "version": "1.0.1",
-    "publisher": "rosshamish",
+    "version": "1.0.2",
+    "publisher": "stevenhurwitt",
```

@stevenhurwitt - I'm still interested in merging this PR if you're willing! I think all that's left to do is reset the publisher name here back to `rosshamish`

```suggestion
    "publisher": "stevenhurwitt",
```